Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description:
 Use keyctl from the current build rather the one from system (Closes: #659299).

diff -Naurp keyutils.orig/tests/prepare.inc.sh keyutils/tests/prepare.inc.sh
--- keyutils.orig/tests/prepare.inc.sh	2012-06-29 20:58:48.768076342 +0200
+++ keyutils/tests/prepare.inc.sh	2012-06-29 20:58:15.439809194 +0200
@@ -1,10 +1,12 @@
 # preparation script for running keyring tests
 
+alias keyctl="../../../../keyctl"
+
 # --- need to run in own session keyring
 if [ "x`keyctl rdescribe @s | sed 's/.*;//'`" != "xRHTS/keyctl/$$" ]
 then
     echo "Running with session keyring RHTS/keyctl/$$"
-    exec keyctl session "RHTS/keyctl/$$" sh $0 $@ || exit 8
+    keyctl session "RHTS/keyctl/$$" sh $0 $@ || exit 8
 fi
 
 # Set up for the Red Hat Test System
